### PR TITLE
adding login page functionality in both HTML & Python program.

### PR DESCRIPTION
**Pull request by Reebs296**

### CODE REVIEW

File: src/flaskr/loginPage.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        return redirect(url_for('login_page'))
+
+
+@app.route('/success')
+def success():
+    return "Login successful!"
+
+
+if __name__ == '__main__':
+    app.run(debug=True)
```
Comment (JamesShadowfyre) [inline src/flaskr/loginPage.py:38]:
```suggestion
    app.run()
```

**Review by JamesShadowfyre — CHANGES_REQUESTED** [review]
Make this change to prevent security vulnerability

File: src/flaskr/loginPage.py (hunk 1 above)
Comment (Reebs296) [reply src/flaskr/loginPage.py:38]: I have now made the changes.

**Review by JamesShadowfyre — APPROVED** [review]
Looks good on HTML end of things. We will need to add an issue to rework to work with database for next week. Good to merge as is

**Review by Birdbh — COMMENTED** [review]
HTML looks good, a little confused by the LoginPage.py as this is already implimented in auth.py and __init__.py , No conflicts from commit but it will be duplicate code

**Pull request merged**